test/Examples.hs

```diff
@@ -0,0 +1,7 @@
+module Examples where
+
+import Base
+import Examples.Positive qualified as P
+
+allTests :: TestTree
+allTests = testGroup "Juvix compilation pipeline tests" [P.allTests]
```

Comment:
There should be a different test group name, e.g., "Juvix milestone examples compilation tests"